docker/build_helper.sh

```diff
@@ -134,7 +126,18 @@ if [ "${BUILD_PYTHON}" = "ON" ]; then
         ### `--set-interpreter' with a larger path than the original is used).
         ### This appears to be a bug in binutils
         ### (http://bugs.strategoxt.org/browse/NIXPKGS-85).
+        bundle_wheel nvidia_dali[_-]*.whl NO ${OUT_DEBUG_WHL_NAME} "${BUNDLE_PATH_PREFIX}" &
+        bundle_wheel nvidia_dali[_-]*.whl YES ${OUT_WHL_NAME} "${BUNDLE_PATH_PREFIX}" &
+        wait
+    else
+        bundle_wheel nvidia_dali[_-]*.whl NO ${OUT_WHL_NAME} "${BUNDLE_PATH_PREFIX}" &
+        wait
+    fi
 
-        ../dali/python/bundle-wheel.sh nvidia_dali[_-]*.whl  ${BUNDLE_PATH_PREFIX}
+    if [ "${TEST_BUNDLED_LIBS}" = "YES" ]; then
+        export UNZIP_PATH="$(mktemp -d)"
+        unzip /wheelhouse/$(ls -I '*_debug.whl' /wheelhouse/) -d $UNZIP_PATH
+        python ../tools/test_bundled_libs.py $(find $UNZIP_PATH -iname *.so* | tr '\n' ' ')
```

Comment:
How long does it take? Could this be run in parallel? Either here or in `test_bundled_libs.py` (I think that even the gelded Python threads could be of use when launching an external program to do the work).

Reply:
It is ~20 whole, and 95% of it is taken by `unzip`.

Reply:
I can make it part of bundle-wheel.sh.

Reply:
Moved.